### PR TITLE
add FanTop address

### PR DESCRIPTION
**Pull request by sone-md**
Thank you always for your support. 
Following the content here, I have created a PR to add our service. I'd be happy if you could check it.

https://docs.blocto.app/blocto-sdk/javascript-sdk/flow/verified-transaction

Our Service FanTop
https://fantop.jp/
We are currently in the final stages of preparing for the transition to v2.

### CODE REVIEW

**Comment by scottphc** [comment]
Hi @sone-md, please add your all related transactions and messages to this project. You can refer to https://github.com/blocto/flow-transactions/pull/43/files

**Comment by sone-md** [comment]
@scottphc 
Thank you for your confirmation. I've added a transaction, so please check it. 
Since there are multiple testnet environments, I've decided to exclude them.

**Comment by scottphc** [comment]
@sone-md you also need to add the message https://github.com/blocto/flow-transactions/pull/43/files#diff-28752302d8186f6037fcf0d341602d6d5930f4465fd029b6f7b87eb5eed635eaR1-R9. It will show the localized message to users for better user experience. 

For example, [transactions/FanTop/transfer_token.json](https://github.com/blocto/flow-transactions/pull/47/files#diff-5cda58d81b7d354f8da82a7aba8cbbbb19237a1ece36d549a9d9ed4d82704ea9) can be:
```
{
  "arguments": ["id", "to"],
  "messages": {
    "en": "Transfer FanTop NFT to {to}",
    "ja": "YOUR JAPANESE TRANSLATION"
  }
}
```

**Comment by sone-md** [comment]
@scottphc 
How is this "message" displayed? If there are any documents or samples, please teach me. I set it at the time of transaction execution, but I could not confirm the display.

```
fcl.mutate({
      message: "FanTopトークンを格納するための保管庫を作成します",
      cadence,
      proposer: fcl.currentUser,
      limit: 1000,
    });
```

**Comment by scottphc** [comment]
@sone-md I mean you can update this file [transactions/FanTop/transfer_token.json](https://github.com/blocto/flow-transactions/pull/47/files#diff-5cda58d81b7d354f8da82a7aba8cbbbb19237a1ece36d549a9d9ed4d82704ea9)

```
{
  "arguments": ["id", "to"],
  "messages": {
    "en": "Transfer FanTop NFT to {to}",
    "ja": "YOUR JAPANESE TRANSLATION"
  }
}
```

And once we merge and update it, the message `Transfer FanTop NFT to 0x1234567890` will show up in the Blocto wallet.

**Comment by sone-md** [comment]
@scottphc I've added a message. How does it look with this kind of image?

File: transactions/FanTop/initialize_collection.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "arguments": ["acct"],
+  "messages": {
+    "en": "I will create a storage vault to store FanTop's items.",
```
Comment (scottphc) [inline transactions/FanTop/initialize_collection.json:4]: Starting with the verb is better, just like other json file you wrote.
```suggestion
    "en": "Create a storage vault to store FanTop's items.",
```

**Review by scottphc — CHANGES_REQUESTED** [review]
LGTM, just minor update.

**Review by scottphc — APPROVED** [review]

**Pull request merged**